### DISCUSSION TITLE
netdata-unified-plans/

### DISCUSSION THREAD

**Discussion by giscus** [netdata/blog#362] [General]
# netdata-unified-plans/

Netdata is introducing a streamlined plan structure, sunsetting Early Bird plans on 13-03-2024.

https://blog.netdata.cloud/netdata-unified-plans/?utm_campaign=website&utm_medium=email&utm_source=sendgrid.com

**Comment by kddlb** [comment]
Lovely. Open core. Does anyone have truly free alternatives?

**Comment by abrindeyev** [comment]
Unless there's a full self-hosted option is available, I'll remove Netdata completely from my homelab - doesn't worth it. Grafana + Prometheus should be enough for my needs - and it's currently free.

The desire to charge *per node* for data that is sitting on my servers is silly.

**Reply by phiberoptick** [reply]
Well... Yes and no.  If you don't want/need their cloud dashboard and never use it, then I see your point.

However, the bandwidth used to feed those metrics in and serve them back out is not miniscule.  Sure, for a single small deployment.  But it all add up.  With the user base they have now, I wouldn't doubt that they pay more in hosting and infrastructure a month than many make in a year.

Besides the bandwidth, they still need compute and resources to actually host the dashboards and the database clusters.  In order to have the resources available and scalability, they would be forced to reserve it.  Meaning they are paying for it if it is used or not.  They just pay a lot more when it is.  All of it is needed to continue serving up snappy and useful metrics to who knows how many end points.  Probably just dedicating a window to the dashboard...so it never stops....

I get that having something "free" and you use or rely on, just go away or start charging.  But  they have been basically giving it away since the beginning.

Without capitol, any project that gets large, will be doomed to fail or halt any innovation.  That's just how it is.  It can't be free forever.  Besides, for most, the cost of spinning up a VM or server to run a parent node, with all the storage, memory, and compute to be capable of handling hundreds or thousands of child nodes reliably and retaining the data for longer than a few hours, I suspect it would cost MANY MANY times what their subscription fees will be.

If it is important and crucial enough to get upset, perhaps the value for you is there?  I just stings a bit to be *forced* to do anything.  Why care what happens or enough to comment otherwise imo.

As I said, for home labs and smaller deployments the cost/value may not be there.  However, I guarantee their funding/business model does not rely on (and never could) the gratitude and charity of others and hope that they would be willing to donate.  While many would.  Most are more happy using it for free.  I don't blame them, that's most likely one of, if not the biggest factor in them choosing it to begin with.

Just my 2

**Comment by phiberoptick** [comment]
Didn't mean to drop that as a random reply to a comment to reposting here.  *Sorry*

Well... Yes and no. If you don't want/need their cloud dashboard and never use it, then I see your point.

However, the bandwidth used to feed those metrics in and serve them back out is not miniscule. Sure, for a single small deployment. But it all add up. With the user base they have now, I wouldn't doubt that they pay more in hosting and infrastructure a month than many make in a year.

Besides the bandwidth, they still need compute and resources to actually host the dashboards and the database clusters. In order to have the resources available and scalability, they would be forced to reserve it. Meaning they are paying for it if it is used or not. They just pay a lot more when it is. All of it is needed to continue serving up snappy and useful metrics to who knows how many end points. Probably just dedicating a window to the dashboard...so it never stops....

I get that having something "free" and you use or rely on, just go away or start charging. But they have been basically giving it away since the beginning.

Without capitol, any project that gets large, will be doomed to fail or halt any innovation. That's just how it is. It can't be free forever. Besides, for most, the cost of spinning up a VM or server to run a parent node, with all the storage, memory, and compute to be capable of handling hundreds or thousands of child nodes reliably and retaining the data for longer than a few hours, I suspect it would cost MANY MANY times what their subscription fees will be.

If it is important and crucial enough to get upset, perhaps the value for you is there? I just stings a bit to be forced to do anything. Why care what happens or enough to comment otherwise imo.

As I said, for home labs and smaller deployments the cost/value may not be there. However, I guarantee their funding/business model does not rely on (and never could) the gratitude and charity of others and hope that they would be willing to donate. While many would. Most are more happy using it for free. I don't blame them, that's most likely one of, if not the biggest factor in them choosing it to begin with.

Just my 2

**Comment by exetico** [comment]
Am I the only one feeling that Netdata Inc. does not touch the "Open Source" touchpoint, and that's the actual change for that part? I'm rocking Netdata on most of my servers, but, I'm not really with If I should expect any changes, if I'm only running it locally, without having my instances connected to the Netdata Cloud service.

**Comment by Hufschmidt** [comment]
90$ per year for "homelab and students" sounds like a rather "steep" price to me, that money I'd most certainly not be spending as a student and that sounds even more rediculous to pay as a homelab user where apart from "doing you own stuff" its also about keeping external costs low.

And on top of that, comparing this to costs for renting a decent VPS with unlimited bandwith costs you around 65€ (60$) per year, this sounds like a total ripoff.
